### ISSUE TITLE
Anime import not working correctly

### ISSUE THREAD

**Issue by Francoisdsr** [Sonarr/Sonarr#4932]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Let's say i want to download the episode 1013 of One Piece.
Within sonarr, the episode 1013 is also note as S21E122
First, I only have the notification that S21E122 (1013) is downloading while it is, and once the episode finish download, sonarr will import it as S10E13 (the relative path is : One Piece - S10E13 - 0208 - [Title]WEBDL-1080p.mkv if that can help)

### Expected Behavior

What i would like to happened is : 
If I grab the 1013 for the S21E122 (since it is note as this in sonarr), that it import it to S21E122 and not S10E13

### Steps To Reproduce

Set an indexers in sonarr that as this option enabled (in my case it was yggtorrent) : 
- Enhance sonarr compatibility with anime by renaming episode (xxx to exxx) 
I use prowlarr as indexers and did this modification to made this option worked like this (xxxx to exxxx) : https://github.com/Prowlarr/Prowlarr/issues/863
Add One Piece series and set the type anime
Try to download the S21E122 (1013) and see that the download notification doesn't appear and the file is import as S10E13 et not the right one

### Environment

```markdown
- OS: raspberry pi os 64 bit
- Sonarr: 3.0.6.1342
- Docker Install: linuxserver.io
- Using Reverse Proxy:
- Browser:
```

### What branch are you running?

Main

### Trace Logs?

The logs stop when i download :
022-03-06 14:08:01.1|Info|NzbSearchService|Searching indexers for [One Piece : 1013]. 1 active indexers
2022-03-06 14:08:40.6|Info|DownloadDecisionMaker|Processing 6 releases
2022-03-06 14:09:10.4|Info|DownloadService|Report sent to Transmission-eth. One Piece - E1013 VOSTFR 1080p WEB x264 -NanDesuKa (ADN)

### Anything else?

<img width="1676" alt="Capture d’écran 2022-03-06 à 15 28 48" src="https://user-images.githubusercontent.com/34095323/156927545-ca180c1d-60b0-4e4d-b80e-0937240c5815.png">

**Comment by bakerboy448** [comment]
- info logs are invalid for bug reports; trace logs are required, what was unclear from the bug report template?
- V3.0.6 is out of date, upgrade pretty sure this is already fixed in 3.0.7
-  you've provided no information - specifically no downloaded file name - for this to even be investigated let alone fixed

**Comment by Francoisdsr** [comment]
Alright, the 3.0.7 did fixed it thanks
